harness/determined/tensorboard/base.py

```diff
@@ -14,7 +14,11 @@ class TensorboardManager:
     implements the sync method.
     """
 
-    def __init__(self, base_path: pathlib.Path, sync_path: pathlib.Path):
+    def __init__(
+        self,
+        base_path: pathlib.Path,
+        sync_path: pathlib.Path,
+    ):
```

Comment:
wait... this file isn't being linted, or this `__init__` without a `-> None` would throw an error... right?

Reply:
i'll peek at this.

Reply:
OK, the file is definitely being linted, e.g. if I remove the ` -> None` from `sync`, it flips out, but if I remove it from any `__init__` call it doesn't seem to mind 🙃 ?